src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/service/listeners/events/additionalinfo/PrisonerAlertsUpdatedNotificationInfo.kt

```diff
@@ -9,8 +9,8 @@ data class PrisonerAlertsUpdatedNotificationInfo(
   val nomsNumber: String,
 
   @JsonProperty("alertsAdded")
-  val alertsAdded: List<String>,
+  var alertsAdded: List<String>,
```

Comment:
should this and below variable be renamed to supportedAlerts or something similar?